plugins/inputs/statsd/datadog.go

```diff
@@ -0,0 +1,173 @@
+package statsd
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const (
+	priorityNormal = "normal"
+	priorityLow    = "low"
+)
+
+var uncommenter = strings.NewReplacer("\\n", "\n")
```

Comment:
@danielnelson in datadog events newlines are encoded as \\n (literally 2 characters).  Should this be transformed into line feed (0x0A) for the tag?